TicTacToe-GUI/TicTacToe.py

```diff
@@ -0,0 +1,232 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Apr 10 19:44:45 2021
+
+@author: Lakhan Kumawat
+"""
+
+# -*- coding: utf-8 -*-
+"""
+Tic Toe Using pygame , numpy and sys with Graphical User Interface
+
+"""
+
+import pygame, sys
+from pygame.locals import *
+import numpy as np
+#import WorkingModal as Wp

```

Comment:
remove this comment

Reply:
> Please Ignore first commit consider second one which is just now .
> First Commit had some issues
> All requested changes are solved except one
> 
> > Tie is not detected even if you see i have commited somewhere to check a tie but it's not working rest everything works fine.

Yeah i mentioned it here i will try removing this bug.